docs/maps/maps-aggregations.asciidoc

```diff
@@ -40,6 +40,18 @@ This field will be used in the terms aggregation to group your documents into en
 This field will be used to sort your documents in the top hits aggregation.
 . Set *Documents per entity* to configure the maximum number of documents accumulated per entity.
 
+[role="xpack"]
+[[point-to-point]]
+=== Point to point
+
+Point to point plots aggregated data paths between the source and destination.
+Thicker darker lines symbolize more connections between a destination and source, and thinner lighter lines symbolize less connections between a destination and source.
+
+Point to point uses {es} {ref}/search-aggregations-bucket-terms-aggregation.html[terms aggregation] to group your documents by destination.
+Then, a nested {ref}/search-aggregations-bucket-geotilegrid-aggregation.html[GeoTile grid aggregation] groups sources for each destination into grids.
+A line is displayed connecting each destination to each source grid centroid.
```

Comment:
Elsewhere you mention source first.  Is it ok to say:

A line connects each source grid centroid to each destination.